assets/css/lesson.scss

```diff
@@ -299,6 +299,10 @@ span.fold-unfold {
 }
 
 
+img.image-with-shadow {
+  box-shadow: 0 4px 16px 0 rgba(0, 0, 0, 0.2), 0 6px 24px 0 rgba(0, 0, 0, 0.19);
```

Comment:
This adds two shadows, both of which have no horizontal offset, similar vertical offsets (4 vs 6), different blur (16 vs 24), no spread, and very similar colors (~20% transparent black).

I'd suggest using a single shadow that would mimick the effect of these two shadows. It might also be easier on browsers to use some non-transparent gray color instead of transparent black (e.g. `#888`)

Another comment we could apply this shadow by default to all images in `article` (by changing the selector from `img.image-with-shadow` to `article img`). What do you think?